content/kontakt/members.json

```
@@ -1,41 +1,41 @@
 [
   {
-    "name": "Johannes Skivdal",
-    "email": "johannes.skivdal@fribyte.no",
-    "position": "leder_guru",
-    "joined": "2022-10-04T00:00:00.000Z",
+    "name": "Simen Strømsnes",
+    "email": "zimen@fribyte.no",
```

La til en identitet til migadu eposten, så vil gjerne bruke den istedet.
```suggestion
    "email": "simen.stromsnes@fribyte.no",
```